metal/resource_metal_device.go

```diff
@@ -348,7 +350,61 @@ func resourceMetalDevice() *schema.Resource {
 				Default:     false,
 				ForceNew:    false,
 			},
+			"reinstall": {
```

Comment:
These fields will have to be documented in docs/resources/device.md.

We have some ongoing work to automate this in #13.